src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isError, setError] = useState(false);
+  const [isLoaded, setLoaded] = useState(false);
+
+  useEffect(() => {
+    setLoaded(false);
+
+    const getPeopleFromSever = async () => {
+      try {
+        const peopleFromServer = await getPeople();
+
+        const linkedPeople = peopleFromServer.map(person => {
+          const newPerson = { ...person };
+
+          function findParentByName(
+            parent: Person[],
+            parentName: string | null,
+          ): Person | undefined {
+            return parent.find(human => human.name === parentName);
+          }
```

Comment:
Saves you a couple of lines, arrow functions are preferred for such short functions. Also, no need to specify return type, as it inferred from `.find`.

```suggestion
          const findParentByName = (
            parent: Person[],
            parentName: string | null,
          ) => parent.find(human => human.name === parentName);
```

Reply:
Fixed

Reply:
Fixed